### ISSUE TITLE
[Feat]: We Can edit this navbar 

### ISSUE THREAD

**Issue by himanshu1221** [FrancescoXX/free-Web3-resources#395]
### Description

We can make navbar more beautiful and fix it 

### Solution

I want to either remove the logo or remove this **Free web3 resources** cause it's looking too loaded when we load it in other devices like phone 

### Anything else?

![Screenshot from 2023-02-10 16-14-34](https://user-images.githubusercontent.com/32031706/218072877-b2bb1ae9-4a63-427b-b0b2-b6316cdaf5e1.png)
<br>
![Screenshot from 2023-02-10 16-14-24](https://user-images.githubusercontent.com/32031706/218072886-613b0453-8124-414f-9f74-54716779afb7.png)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/FrancescoXX/4c-site/blob/main/docs/CODE_OF_CONDUCT.md)

**Comment by himanshu1221** [comment]
> ### Description
> We can make navbar more beautiful and fix it
> 
> ### Solution
> I want to either remove the logo or remove this **Free web3 resources** cause it's looking too loaded when we load it in other devices like phone
> 
> ### Anything else?
> ![Screenshot from 2023-02-10 16-14-34](https://user-images.githubusercontent.com/32031706/218072877-b2bb1ae9-4a63-427b-b0b2-b6316cdaf5e1.png) ![Screenshot from 2023-02-10 16-14-24](https://user-images.githubusercontent.com/32031706/218072886-613b0453-8124-414f-9f74-54716779afb7.png)
> 
> ### Code of Conduct
> * [x]  I agree to follow this project's [Code of Conduct](https://github.com/FrancescoXX/4c-site/blob/main/docs/CODE_OF_CONDUCT.md)

<h3>It would be great if you assign this issue to me so that i an start working on it  </h3>

**Comment by tobySolutions** [comment]
Hey @himanshu1221!! We cannot remove the logo, but, I'll assign you this issue to see what you can do with it.

**Comment by tobySolutions** [comment]
Any updates @himanshu1221??

**Comment by himanshu1221** [comment]
> Any updates @himanshu1221??

I was busy in my exams I'll start working on this today !

**Comment by tobySolutions** [comment]
Oh! Didn't know that @himanshu1221. Please, only do this if it will not affect your studying. Thank you so much Himanshu! :rocket:

**Comment by himanshu1221** [comment]
> Oh! Didn't know that @himanshu1221. Please, only do this if it will not affect your studying. Thank you so much Himanshu! rocket

Sure Toby i will work on it but a bit slow but i will ..

**Comment by tobySolutions** [comment]
> > Oh! Didn't know that @himanshu1221. Please, only do this if it will not affect your studying. Thank you so much Himanshu! rocket
> 
> Sure Toby i will work on it but a bit slow but i will ..

Awesome! Sounds cool!

**Comment by himanshu1221** [comment]
> > > Oh! Didn't know that @himanshu1221. Please, only do this if it will not affect your studying. Thank you so much Himanshu! rocket
> > 
> > 
> > Sure Toby i will work on it but a bit slow but i will ..
> 
> Awesome! Sounds cool!

Hey Toby i am not able to commit my changes could you please help me out !
here is the screen shot 

![Screenshot from 2023-02-15 17-15-32](https://user-images.githubusercontent.com/32031706/219019021-b1e332b1-5392-4cb2-a5b0-24b37a100dc8.png)

**Comment by tobySolutions** [comment]
Hey @himanshu1221!! Add a space after the colon like this "feat: your commit message"

**Comment by himanshu1221** [comment]
> Hey @himanshu1221!! Add a space after the colon like this "feat: your commit message"

Hey @tobySolutions  thanks for helping me out 
i have just made a PR's please do review that !

**Comment by tobySolutions** [comment]
Pleasure is all mine @himanshu1221. Sure! I'll check out the PR soon.
